packages/api/src/modules/providers/apiAsso/apiAsso.service.ts

```diff
@@ -184,10 +184,15 @@ export class ApiAssoService
 
     private filterActiveDacDocuments(documents: StructureDacDocumentDto[], structureIdentifier: StructureIdentifiers) {
         if (!Array.isArray(documents)) {
-            const errorMessage = "API-ASSO documents is not an array for structure " + structureIdentifier;
-            Sentry.captureException(new Error(errorMessage));
-            console.error(errorMessage);
-            return [];
+            if ((documents as Record<string, unknown>).uuid !== undefined) {
+                // When api have one document, it is not an array but a single object
+                documents = [documents];
+            } else {
+                const errorMessage = "API-ASSO documents is not an array for structure " + structureIdentifier;
```

Comment:
Le message prête à confusion maintenant

Reply:
un truc genre dac documents not found for structure blabla t'irait ?

Reply:
C'est vraiment ça l'erreur dans ce cas-là ? Si c'était ça on ne devrait même pas rentrer là ?
De ma compréhension c'est plutôt "document in unknwon format" ..? D'ailleurs on peut peut-être l'envoyer dans l'erreur

Reply:
Ok pour toi ?

Reply:
yup